WooCommerce/Classes/ViewRelated/Dashboard/DashboardView.swift

```diff
@@ -73,6 +73,9 @@ struct DashboardView: View {
 
             // Card views
             dashboardCards
+
+            Spacer()
+                .frame(height: Layout.padding)
```

Comment:
Nit: this can be replaced with a bottom padding. Since we want to add padding to the top too, let's put a vertical padding.

```suggestion
                .padding(.vertical, Layout.padding)
```